lib/Test2/API.pm

```diff
@@ -406,7 +406,7 @@ sub _existing_error {
     my $oldframe = $ctx->{trace}->frame;
     my $olddepth = $ctx->{_depth};
 
-    my $mess = longmess();
+    my $mess = Carp::longmess();
```

Comment:
This sort of change really needs a comment or else someone is likely to revert it back later.

Reply:
I added comment also into code via git fixup commit.